generic_config_updater/gcu_field_operation_validators.conf.json

```diff
@@ -10,11 +10,113 @@
+            "validator_data": {
+                "rdma_config_update_validator": {
+                    "PFCWD enable/disable": {
+                        "fields": [
+                            "restoration_time",
+                            "detection_time",
+                            "action"
+                        ],
+                        "operations": ["remove", "add", "replace"],
+                        "platforms": {
+                            "spc1": "20181100",
+                            "td2": "20181100",
+                            "th": "20181100",
+                            "th2": "20181100",
+                            "td3": "20181100",
+                            "cisco-8000": "20201200"
+                        }
+                    }
+                }
+            }
+        },
+        "buffer_pool": {
+            "field_operation_validators": [ "generic_config_updater.field_operation_validators.rdma_config_update_validator" ],
+            "validator_data": {
+                "rdma_config_update_validator": {
+                    "Shared/headroom pool size changes": {
+                        "fields": [
+                            "ingress_lossless_pool/xoff",
+                            "ingress_lossless_pool/size",
+                            "egress_lossy_pool/size"
+                        ],
+                        "operations": ["replace"],
+                        "platforms": {
+                            "spc1": "20191100",
+                            "td2": "",
+                            "th": "20221100",
+                            "th2": "20221100",
+                            "td3": "20221100",
+                            "cisco-8000": ""
+                        }
+                    }
+                }
+            }
+        },
+        "buffer_profile": {
+            "field_operation_validators": [ "generic_config_updater.field_operation_validators.rdma_config_update_validator" ],
+            "validator_data": {
+                "rdma_config_update_validator": {
+                    "Dynamic threshold tuning": {
+                        "fields": [
+                            "dynamic_th"
+                        ],
+                        "operations": ["replace"],
+                        "platforms": {
+                            "spc1": "20181100",
+                            "td2": "20181100",
+                            "th": "20181100",
+                            "th2": "20181100",
+                            "td3": "20201200",
+                            "cisco-8000": ""
+                        }
+                    },
+                    "PG headroom modification": {
+                        "fields": [
+                            "xoff"
+                        ],
+                        "operations": ["replace"],
+                        "platforms": {
+                            "spc1": "20191100",
+                            "td2": "",
```

Comment:
>"td2": "", [](http://example.com/codeflow?start=28&length=10)

I see "NA" and "Unknown" in the table, but they are both implemented as "". Is it a concern?

Reply:
It should not be a concern because we agreed that both "NA" and "Unknown" are considered unsupported. If the code reads "", we return False because this change is unsupported